kedro-telemetry-requirements.txt

```diff
@@ -0,0 +1,2 @@
+packaging
+kedro-telemetry>=0.6.0 # First version that does not prompt for telemetry
```

Comment:
We need to add `appdirs` as one more dependency for `kedro-telemetry`. As we are not installing all its required dependencies. I got the importError for `appdirs` when I was testing.

Reply:
Nice found